ush/prepobs_makeprepbufr.sh

```diff
@@ -2114,7 +2117,8 @@ set -x
          elif [ "$launcher_PREP" = mpiexec ]; then
 	    chmod 755 $DATA/prep_exec.cmd
             #mpiexec -n 6 -ppn 32 $DATA/prep_exec.cmd #IG
-             mpiexec -n 6 $DATA/prep_exec.cmd
+            # mpiexec -n 6 $DATA/prep_exec.cmd
+            mpiexec -n $NSPLIT cfp $DATA/prep_exec.cmd  
```

Comment:
@ilianagenkova please check our test run. Your test run is not using this new code. Your $HOMEobsproc is set to `/u/Iliana.Genkova/INSTALLpre/obsproc` and the `ush/prepobs_makeprepbufr.sh` there does not have Jack's changes in it. Can you re-run and be sure you are running w/ his changes? For reference, I am looking in `/lfs/h2/emc/ptmp/Iliana.Genkova/obsproc_gfs_prep_JWool_20210824_06.o9937950`